README.md

```diff
@@ -67,6 +67,7 @@ this repository may be used on the following networks:
       - *double-check that wasm-bindgen-cli version above matches wasm-bindgen version in Cargo.lock file*
       - *Depending on system, additional packages may need to be installed as a prerequisite for wasm-bindgen-cli. If anything is missing, installation will error and prompt what packages are missing (i.e. clang, llvm, libssl-dev)*
   - essential build tools - example for Debian/Ubuntu: `apt install -y build-essential libssl-dev pkg-config clang llvm`
+  - [trunk](https://trunkrs.dev/) for GroveDBG support
```

Comment:
Is it possible to do it somehow optionally. In most of cases you devs don't GroveDBG

Reply:
Even though grovedbg is under a feature flag and totally optional, Drive abci is an executable as far as I know, and that means we should compile different versions of the platform depending on the configuration to apply feature flags

Reply:
Yes, let's release 2 versions.

Reply:
I can add a feature flag to rs-drive-abci but i'm not sure how to proceed next

Reply:
We need to manage it in dockerfile then. using build args. So if we pass a specific build arg we enable this feature.

Reply:
could you please provide more details on how can I achieve that?

visualizer is under the feature gate already, I assume for CI `trunk` will be needed anyways because we test all features, for the docker part i'm not sure, just an `ARG` to build drive abci either with that feature or not?

Reply:
Trunk is no longer used